dotcom-rendering/cypress/e2e/parallel-1/sign-in-gate.cy.js

```diff
@@ -59,7 +59,7 @@ describe('Sign In Gate Tests', function () {
 		beforeEach(function () {
 			disableCMP();
 			// sign in gate main runs from 0-900000 MVT IDs, so 500 forces user into test
-			setMvtCookie('500');
+			setMvtCookie('500000');
```

Comment:
For readability, this is identical in modern JS:

```suggestion
			setMvtCookie('500_000');
```

Reply:
🤯